e2e/tests/visual.spec.ts

```diff
@@ -116,6 +116,7 @@ test.describe("Share functionality", () => {
 
     await page.click(".monaco-editor")
```

Comment:
I think you could avoid flakiness simply by waiting for network to be idle.

Reply:
networkidle is strongly discouraged: https://playwright.dev/docs/api/class-page#page-wait-for-load-state